### PR TITLE
chore: apply whitespace fixes to component assets

### PR DESCRIPTION
**Pull request by castastrophe**
## Description

This PR applies whitespace, logical property rename, and opacity conversions to all provided assets. The eslint changes applied include newlines after curly braces and converting single to double quotes in *.js files.

## How and where has this been tested?

Please tag yourself on the tests you've marked complete to confirm the tests have been run by someone other than the author.

### Validation steps

- [x] Expect to see no VRT regressions.

### Regression testing

Validate:

1. The documentation pages for at least two other components are still loading, including:

- [ ] The pages render correctly, are accessible, and are responsive.

2. If components have been modified, VRTs have been run on this branch:

- [ ] VRTs have been run and looked at.
- [ ] Any VRT changes have been accepted (by reviewer and/or PR author), or there are no changes.

## To-do list

- [x] I have read the [contribution guidelines](/.github/CONTRIBUTING.md).
- [ ] I have updated relevant storybook stories and templates.
- [ ] I have tested these changes in Windows High Contrast mode.
- [ ] If my change impacts **other components**, I have tested to make sure they don't break.
- [ ] If my change impacts **documentation**, I have updated the documentation accordingly.
- [x] ✨ This pull request is ready to merge. ✨

### CODE REVIEW

File: components/calendar/index.css [hunk 1]
```diff
@@ -141,14 +141,12 @@ governing permissions and limitations under the License.
...
 .spectrum-Calendar-table {
   table-layout: fixed;
-  /*width: 100%;*/
+
+  /* width: 100%; */
 
   border-collapse: collapse;
   border-spacing: 0;
 
-  -webkit-user-select: none;
```
Comment (castastrophe) [inline components/calendar/index.css:149]: https://caniuse.com/mdn-css_properties_user-select

File: components/dial/index.css [hunk 2]
```diff
@@ -231,13 +234,14 @@ governing permissions and limitations under the License.
   overflow: hidden;
   opacity: 0.000001;
   cursor: default;
-  -webkit-appearance: none;
```
Comment (castastrophe) [inline components/dial/index.css:234]: Support good on this one - no need to prefix anymore

File: components/link/index.css [hunk 3]
```diff
@@ -51,8 +51,9 @@ governing permissions and limitations under the License.
 .spectrum-Link {
   /* Remove the gray background on active links in IE 10. */
   background-color: transparent;
+
   /* Remove gaps in links underline in iOS 8+ and Safari 8+. */
-  -webkit-text-decoration-skip: objects;
+  text-decoration-skip: objects;
```
Comment (castastrophe) [inline components/link/index.css:56]: auto-prefixer is adding the -webkit- prefix for us post-processing

**Comment by pfulton** [comment]
@castastrophe the storybook build was failing here due to a wonky import for a react hook, after our move to SB8. I pushed up [a fix](https://github.com/adobe/spectrum-css/pull/2640/commits/b6440de096244e9ffe52c8d83526fda15337c6d6) for this.

**Pull request merged**